meerk40t/gui/wxmmain.py

```diff
@@ -554,7 +555,26 @@ def sub_register(kernel):
                 "size": buttonsize
             },
         )
-
+        kernel.register(
+            "button/modify/Rotate90CW",
+            {
+                "label": _("Rotate CW"),
+                "icon": icons8_rotate_right_50,
+                "tip": _("Rotate the selected element clockwise by 90 deg"),
+                "action": lambda v: kernel.elements("rotate 90deg\n"),
+                "size": buttonsize
+            },
+        )
+        kernel.register(
+            "button/modify/Rotate90CCW",
+            {
+                "label": _("Rotate CCW"),
+                "icon": icons8_rotate_left_50,
+                "tip": _("Rotate the selected element counterclockwise by 90 deg"),
+                "action": lambda v: kernel.elements("rotate -90deg\n"),
+                "size": buttonsize
+            },
+        )
```

Comment:
Perfect fine addition, might prefer a different PR.

Reply:
Apologies, was working on a different PR but VS Code was still using the old branch